acceptance-tests.md

```diff
@@ -114,8 +114,12 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Establish OpenShift cluster upgrade process**
     * _As an administrator of the cluster, I should be able to follow a set of documented instructions to help me in upgrading to newer versions of OpenShift. The rule book should also establish the process to mitigate any issues that might arise during the upgrade._
-    * <span style="text-decoration:underline;">Steps</span>
-        1. Upgrade documentation available in the documentation repository
+    * Steps
+        1. See the official [OpenShift updating clusters documentation here](https://docs.openshift.com/container-platform/4.10/updating/index.html) for the version of OpenShift you wish to migrate to.
```

Comment:
Remove `here` in the link and a minor grammar fix:

```suggestion
        1. See the official [OpenShift updating clusters documentation](https://docs.openshift.com/container-platform/4.10/updating/index.html) for the version of OpenShift to which you wish to upgrade.
```

Reply:
@larsks Thanks for the suggestion, I have made this change and fixed quotes.